CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
+ * Support `xml.format.joinCommentLines` setting with experimental formatter. See [lemminx#1243](https://github.com/eclipse/lemminx/issues/1243).
+ * Support `xml.format.preserveEmptyContent` setting with experimental formatter. See [lemminx#1242](https://github.com/eclipse/lemminx/issues/1242).
+ * Support `xml.format.joinCDATALines` setting with experimental formatter. See [lemminx#1241](https://github.com/eclipse/lemminx/issues/1241).
+ * Support `xml.format.preservedNewlines` setting with experimental formatter. See [lemminx#1240](https://github.com/eclipse/lemminx/issues/1240).
+ * Support `xml.format.enforceQuoteStyle` setting with experimental formatter. See [lemminx#1239](https://github.com/eclipse/lemminx/issues/1239).
+ * Support `xml.format.emptyElements` setting with experimental formatter. See [lemminx#1238](https://github.com/eclipse/lemminx/issues/1238).
+ * Set `xml.format.xsiSchemaLocationSplit` as onPair by default. See [#749](https://github.com/redhat-developer/vscode-xml/issues/749).
+ * Provide comment formatting for experimental formatter. See [lemminx#1259](https://github.com/eclipse/lemminx/issues/1259).
+ * Suppress validation kinds based on file pattern. See [#774](https://github.com/redhat-developer/vscode-xml/pull/774).
+ * Report only XML syntax error for *.exsd files. See [#782](https://github.com/redhat-developer/vscode-xml/pull/782). 
+ * Place relevant information on the first line of error messages. See [lemminx#1145](https://github.com/eclipse/lemminx/issues/1145).
+ * Validate uri of XML catalog. See [lemminx#823](https://github.com/eclipse/lemminx/issues/823).
+ * Improve error message for download error. See [#763](https://github.com/redhat-developer/vscode-xml/issues/763). 
+ * CodeAction/Completion to insert required attributes from XSD. See [lemminx#365](https://github.com/eclipse/lemminx/issues/365).
+ * CodeAction for `cvc-complex-type.2.4.b`: `insert all expected` vs. `insert all possible`. See [lemminx#1255](https://github.com/eclipse/lemminx/issues/1255).
+ * Do not complete paths in attr unless beginning of value looks like a path. See [lemminx#1293](https://github.com/eclipse/lemminx/pull/1293).
+ * Add command `Restart Language Server`. See [#539](https://github.com/redhat-developer/vscode-xml/issues/539). 
+ * Use the HTTP proxy configuration for HTTPS as well. See [lemminx#1253](https://github.com/eclipse/lemminx/pull/1253).
+ * Add `usesOnlineServices` tag. See [#701](https://github.com/redhat-developer/vscode-xml/pull/701).
+ * Replace `find-java-home` with `jdk-utils`. See [#627](https://github.com/redhat-developer/vscode-xml/issues/627).
+
+### Bug Fixes
+
+ * Autocompletion of attribute values won't display documentation. See [#736](https://github.com/redhat-developer/vscode-xml/issues/736).
+ * Null pointer exception when processing code action `cvc_complex_type_2_4_b`. See [lemminx#1280](https://github.com/eclipse/lemminx/issues/1280).
```

Comment:
This can be removed also. the code action for 2_4_b was introduced in this release so there's no need to mention we fixed a bug in it, because the users would never see the bug between 0.21.0 and 0.22.0.

Reply:
removed